sympy/solvers/tests/test_solveset.py

```diff
@@ -931,3 +932,15 @@ def test_issue_9557():
 
     assert solveset(x**2 + a, x, S.Reals) == Intersection(S.Reals,
         FiniteSet(-sqrt(-a), sqrt(-a)))
+
+
+def test_issue_9778():
+    assert solveset(x**3 + 1, x, S.Reals) == FiniteSet(-1)
+    assert solveset(x**(S(3)/5) + 1, x, S.Reals) == S.EmptySet
```

Comment:
@hargup  my point is: Finding the root of an equation involving an `nth-power` and finding the `nth-root` of `any-number` are a little different things.